wolfcrypt/src/wc_port.c

```diff
@@ -126,6 +127,161 @@ int wolfCrypt_Cleanup(void)
     return ret;
 }
 
+#if !defined(NO_FILESYSTEM)
+
+int wc_ChangeDir(const char* path)
+{
+    int ret;
+#ifdef USE_WINDOWS_API
+    ret = SetCurrentDirectoryA(path);
+#else
+    ret = chdir(path);
```

Comment:
@toddouska: Just pushed a new rebased / squash merged branch with the ChangeDir stuff reverted. Thanks!